conda/plan.py

```diff
@@ -383,8 +383,8 @@ def add_defaults_to_specs(r, linked, specs, update=False):
             # if Python/Numpy is already linked, we add that instead of the
             # default
             log.debug('H3 %s' % name)
-            fkey = names_linked[name] + '.tar.bz2'
-            info = r.index[fkey]
+            fkey = names_linked[name]
+            info = r.index[fkey if fkey.endswith('.tar.bz2') else fkey + '.tar.bz2']
```

Comment:
Actually, `fkey` needs to end in `.tar.bz2` later in the code. So you need to do this:
```
if not fkey.endswith('.tar.bz2'):
    fkey += '.tar.bz2'
```
or something like that